pkg/storer/reserve.go

```diff
@@ -500,23 +500,27 @@ func (db *DB) SubscribeBin(ctx context.Context, bin uint8, start uint64) (<-chan
 type NeighborhoodStat struct {
 	Neighborhood            swarm.Neighborhood
 	ReserveSizeWithinRadius int
+	Proximity               uint8
 }
 
 func (db *DB) NeighborhoodsStat(ctx context.Context) ([]*NeighborhoodStat, error) {
 
 	radius := db.StorageRadius()
 
-	networkRadius := radius + uint8(db.reserveOptions.capacityDoubling)
+	responsibilityRadius := radius + uint8(db.reserveOptions.capacityDoubling)
```

Comment:
Can `radius + uint8(db.reserveOptions.capacityDoubling)` be bigger then 255?

Reply:
should not be possible